.travis.yml

```
@@ -1,3 +1,9 @@
+branches:
+  only:
+  - master
+  - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
```

if we don't need to be strict with version names, this could be simplified as 
```suggestion
  - /^v\d+.*$/
```

---

Yes, but if we leave the first regexp, it can serve as check to avoid tag builds if we make a mistake with the semver. e.g. a tag of `v1.2..3` will not be run on travis.